src/OpenSpaceToolkit/Astrodynamics/Dynamics/Tabulated.cpp

```diff
@@ -105,6 +107,17 @@ Shared<const Frame> Tabulated::getFrame() const
     return accessFrame();
 }
 
+Array<Interpolator::Type> Tabulated::getInterpolationTypes() const
+{
+    Array<Interpolator::Type> interpolationTypes = Array<Interpolator::Type>::Empty();
+
+    for (auto& interpolationSPtr : interpolators_)
```

Comment:
```suggestion
    for (const auto& interpolationSPtr : interpolators_)
```
Always const unless we are modifying it.

Reply:
Ah yes good point, even more important when dealing with references